tests/Mapper/ProgrammesDbToDomain/CoreEntityMapperOptionsMappingTest.php

```diff
@@ -27,7 +27,7 @@ public function testGetDomainModelHierarchy()
 
         $expectedMasterBrandDomainEntity = $this->createMock(MasterBrand::class);
 
-        $this->mockMasterBrandMapper->expects($this->once())
+        $this->mockMasterBrandMapper->expects($this->atLeastOnce())
```

Comment:
This test contains a programme with a parent. Now that "try look up your masterbrand from your parent" is the default, then the masterBrand mapper is called twice - one for the itself, and one for the parent element.